quick_tour/flex_recipes.rst

```diff
@@ -0,0 +1,249 @@
+Flex: Compose your Application
+==============================
+
+After reading the first part of this tutorial, you have decided that Symfony was
+worth another 10 minutes. Great choice! In this second part, you'll learn about
+Symfony Flex: the amazing tool that makes adding new features as simple as running
+one command. It's also the reason why Symfony is perfect for a small micro-service
```

Comment:
Just asking: can this sound arrogant? --> *"[...] Symfony is perfect for [...]"* Could we reword it?